tests/tests_integration/test_api/test_transformations/test_transformations.py

```diff
@@ -9,9 +9,8 @@
 
 @pytest.fixture
 def new_datasets(cognite_client):
-    prefix = "".join(random.choice(string.ascii_letters) for i in range(6))
-    ds_ext_id1 = f"{prefix}-transformation-ds"
-    ds_ext_id2 = f"{prefix}-transformation-ds2"
+    ds_ext_id1 = "transformation-ds"
```

Comment:
no need to create a new data set every time with random values, it is just upsert.